### ISSUE TITLE
Is there any upload limit?

### ISSUE THREAD

**Issue by ashish453** [tomgross/pycloud#25]
while uploading a file of size 2.65 GB using python I get this error
```
/usr/lib/python3.6/ssl.py in write(self, data)
    640         The 'data' argument must support the buffer interface.
    641         """
--> 642         return self._sslobj.write(data)
    643 
    644     def getpeercert(self, binary_form=False):

OverflowError: string longer than 2147483647 bytes
```
How can i solve this problem? please help...

**Comment by tomgross** [comment]
Hi @ashish453 good catch! Obviously there is a limit with the underlying SSL library (see https://github.com/psf/requests/issues/2717).

We have to investigate, how sending the file handler instead of the file goes together with the pCloud API.

**Comment by tomgross** [comment]
Upload limit has been fixed with 1.0b1: https://pypi.org/project/pcloud/1.0b1/
